src/main/resources/org/sagebionetworks/web/client/widget/table/v2/TotalVisibleResultsWidgetViewImpl.ui.xml

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+			 xmlns:g="urn:import:com.google.gwt.user.client.ui"
+			 xmlns:c="urn:import:com.google.gwt.user.cellview.client"
+			 xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
+			 xmlns:t="urn:import:org.sagebionetworks.web.client.widget.table.v2"
+			 xmlns:bt="urn:import:org.sagebionetworks.web.client.view.bootstrap.table"
+			 xmlns:a="urn:import:org.sagebionetworks.web.client.widget.table.v2"
+			 xmlns:bg="urn:import:org.gwtbootstrap3.client.ui.gwt"
+			 xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html"
+			 xmlns:w="urn:import:org.sagebionetworks.web.client.widget">
+<bh:Div visible="false">
+		<b:Heading size="H3" ui:field="totalItemCountContainer">
+			<bh:Span ui:field="totalItemCountItemName">Files</bh:Span>
+			<bh:Span>&nbsp;(</bh:Span>
+			<bh:Span ui:field="totalItemCount">10000</bh:Span>
```

Comment:
Doesn't matter in this case (since this widget is not shown before it's initialized with data), but I'd prefer this to default to an unknown value (like blank, or '-') rather than a random number.